core/tracepoint/new_spec.rb

```diff
@@ -0,0 +1,55 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+module ClassSpecs
+  class A
+    def foo; end
+  end
+end
+
+def test; 'test' end
+
+ruby_version_is '2.0' do
+  describe 'TracePoint.new' do
+    it 'returns a new TracePoint object, not enabled by default' do
+      TracePoint.new(:call) {}.enabled?.should be_false
+    end
+
+    it 'includes :line event when event is not specified' do
+      event_name = nil
+      TracePoint.new() { |tp| event_name = tp.event }.enable
+      event_name.should equal(:line)
+
+      event_name = nil
+      test
+      event_name.should equal(:line)
+
+      event_name = nil
+      ClassSpecs::A.new.foo
+      event_name.should equal(:line)
+    end
+
+    it 'includes the matching event name if argument is string' do
+      event_name = nil
+      TracePoint.new('return') { |tp| event_name = tp.event}.enable
```

Comment:
To make this even better, instead of a `String` we should pass an object that defines `to_str` or `to_sym` (I'm not sure which is used). See `core/module/method_defined_spec.rb`

Reply:
okay, I have updated the spec